Message locale can be null

**Describe the bug**
Message locale can be null. Some users has null locale. In that case default locale shall be used.

Exception example:

```log
Caused by: java.lang.NullPointerException: Cannot invoke "String.length()" because "<parameter1>" is null
	at java.base/sun.util.locale.LocaleUtils.toLowerString(Unknown Source)
	at java.base/sun.util.locale.LanguageTag.parse(Unknown Source)
	at java.base/java.util.Locale.forLanguageTag(Unknown Source)
	at ru.dankoy.telegrambot.core.service.localeprovider.LocaleProviderImpl.getLocale(LocaleProviderImpl.java:17)
	at ru.dankoy.telegrambot.core.service.bot.commands.CommandsExtractorServiceImpl.getCommand(CommandsExtractorServiceImpl.java:31)
	at ru.dankoy.telegrambot.core.service.flow.BotCommandValidatorImpl.isValid(BotCommandValidatorImpl.java:29)
```

**Expected behavior**
Class BotCOmmandValidator shall check locale on null, if true, get default locale. Or should it be implemented in LocaleProvider?

--- a/build.gradle
+++ b/build.gradle
@@ -6,7 +6,7 @@ plugins {
 }
 
 ext {
-    set("PROJECT_VERSION", "1.3.2")
+    set("PROJECT_VERSION", "1.3.3")
 }
 
 // doesn't work in build.gradle in buildSrc project

--- a/telegram_bot/src/main/java/ru/dankoy/telegrambot/core/service/localeprovider/LocaleProviderImpl.java
+++ b/telegram_bot/src/main/java/ru/dankoy/telegrambot/core/service/localeprovider/LocaleProviderImpl.java
@@ -1,19 +1,37 @@
 package ru.dankoy.telegrambot.core.service.localeprovider;
 
 import java.util.Locale;
+import java.util.Objects;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 import org.telegram.telegrambots.meta.api.objects.Message;
+import ru.dankoy.telegrambot.config.bot.properties.LocaleConfig;
 
 @Service
+@RequiredArgsConstructor
 public class LocaleProviderImpl implements LocaleProvider {
+
+  private final LocaleConfig localeConfig;
 
   @Override
   public Locale getLocale(Message message) {
-    return Locale.forLanguageTag(message.getFrom().getLanguageCode());
+
+    var langCode = message.getFrom().getLanguageCode();
+
+    if (Objects.isNull(langCode)) {
+      return localeConfig.getDefaultLocale();
+    } else {
+      return Locale.of(langCode);
+    }
   }
 
   @Override
   public Locale getLocale(String locale) {
-    return Locale.forLanguageTag(locale);
+
+    if (Objects.isNull(locale)) {
+      return localeConfig.getDefaultLocale();
+    } else {
+      return Locale.of(locale);
+    }
   }
 }

--- a/telegram_bot/src/main/java/ru/dankoy/telegrambot/core/service/localeprovider/LocaleProviderImpl.java
+++ b/telegram_bot/src/main/java/ru/dankoy/telegrambot/core/service/localeprovider/LocaleProviderImpl.java
@@ -18,16 +18,16 @@ public class LocaleProviderImpl implements LocaleProvider {
 
     var langCode = message.getFrom().getLanguageCode();
 
-    if (Objects.isNull(langCode)) {
-      return localeConfig.getDefaultLocale();
-    } else {
-      return Locale.of(langCode);
-    }
+    return checkAndReturn(langCode);
   }
 
   @Override
   public Locale getLocale(String locale) {
 
+    return checkAndReturn(locale);
+  }
+
+  private Locale checkAndReturn(String locale) {
     if (Objects.isNull(locale)) {
       return localeConfig.getDefaultLocale();
     } else {

